config/initializers/authtrail.rb

```diff
@@ -0,0 +1 @@
+AuthTrail.geocode = false
```

Comment:
We can implement geocoding incrementally; not a must-have to start recording login activity. My assumption is that we can also geocode old login attempts with a backfill job since we'll have all the IPs.